app/scripts/modules/core/src/presentation/main.less

```diff
@@ -945,14 +945,53 @@ select:invalid {
 }
 
 .section-heading-chevron {
-  transition: transform 0.15s ease;
-  margin-right: 4px;
+  &.left {
+    margin-right: 4px;
+  }
+
+  &.right {
+    margin-left: 4px;
+  }
+  display: flex;
 }
 
-.rotated-90 {
```

Comment:
make sure to check other repos to see if this is used by, e.g., deck-kayenta or deck-nflx